dhis-2/dhis-web-api/src/main/java/org/hisp/dhis/webapi/controller/event/TrackedEntityAttributeController.java

```diff
@@ -91,13 +91,6 @@
         @PathVariable String id )
         throws WebMessageException
     {
-        TrackedEntityAttribute trackedEntityAttribute = trackedEntityAttributeService.getTrackedEntityAttribute( id );
```

Comment:
why are is this removed here and in the method below?

Reply:
every call for reserve values is now passing in _reserve_  private method where we have all relevant checks

Reply:
where is that method? did you make that change in this PR?